docs/documentation.md

```
@@ -0,0 +1,11 @@
+# Documentation Guide
+
+## Sphinx
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)